tmp/p256_dilithium2_CA.crt

```diff
@@ -0,0 +1,88 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Are these tmp files added on purpose?

Reply:
Absolutely no. Sorry. Will revert. Artifact of running the interop tests in parallel to the `commit` :-(